solidity/contracts/bridge/Bridge.sol

```diff
@@ -963,13 +976,17 @@ contract Bridge is Ownable, EcdsaWalletOwner {
     ///         transaction fee that is acceptable in a single moving funds
     ///         transaction. This is a _total_ max fee for the entire moving
     ///         funds transaction.
+    /// @return movingFundsTimeout Time after which the moving funds process
```

Comment:
It is in seconds, right? Worth mentioning it in the docs.

Reply:
See https://github.com/keep-network/tbtc-v2/pull/212/commits/4af794e278898ada7a640d7e6124ef5cef16696c